tools/journal_proto_processor.cpp

```diff
@@ -1200,6 +1205,8 @@ void JournalProtoProcessor::ProcessInOut(
   }
 
   cs_interface_parameters_[descriptor].clear();
+  cs_interface_marshalled_parameters_[descriptor].clear();
```

Comment:
We spell this word "marshaled" because that's what the .Net documentation does (see https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.marshalasattribute?view=net-9.0).